### ISSUE TITLE
mkm.market_place.articles not working properly?

### ISSUE THREAD

**Issue by flirola** [evonove/mkm-sdk#50]
Hello,

First of all, if it is just me improperly using the tool, I apologize.

It seems that function mkm.market_place.articles doesn't work properly in API 2.0

Indeed, this function should be able to take several arguments:
- idProduct
- start
- maxResults

idProduct is specified as product argument.

However, start and maxResults arguments seemed not to be taken into account. This would useful to allow for pagination of results when too many  elements are available (over 1000).

Basically I would call: mkm.market_place.articles(product=idProduct, start=0, maxResults=249)
I would expect the first 249 results to be returned. Instead all are returned.

I imagine the api_map would have to by updated to match format specified here: [https://api.cardmarket.com/ws/documentation/API_2.0:Articles]

Can you please advise on the proper way to use the function or advise on the corrections to be made?

Thanks for the support!

**Comment by flirola** [comment]
After investigation, the function is working Fine. 
The proper way to use it is: mkm.market_place.articles(product=idProduct, params={'start': 0, 'maxResults': 249})
